src/networking.c

```diff
@@ -109,6 +109,9 @@ static void clientSetDefaultAuth(client *c) {
     c->user = DefaultUser;
```

Comment:
we should also check the current `c->user`, the reset command may change user:
`resetCommand`->`clearClientConnectionState`->`clientSetDefaultAuth`.

Reply:
YEAH. remove the original contacts in the `clearClientConnectionState`  and then add new in `clientSetDefaultAuth `